### ISSUE TITLE
v2.3.0 Broken Pretty Print for Dependency Graph

### ISSUE THREAD

**Issue by msymons** [CycloneDX/cyclonedx-maven-plugin#88]
The dependency graph generated by v2.3.0 of the plugin is not prettyprinted, whereas it is in v2.2.0.  This impacts readability.

Example from v2.2.0 JSON:
```
    {
      "ref": "pkg:maven/io.dropwizard/dropwizard-core@1.3.29?type=jar",
      "dependsOn": [
        "pkg:maven/io.dropwizard/dropwizard-util@1.3.29?type=jar",
        "pkg:maven/io.dropwizard/dropwizard-configuration@1.3.29?type=jar",
        "pkg:maven/io.dropwizard/dropwizard-metrics@1.3.29?type=jar",
        "pkg:maven/io.dropwizard/dropwizard-servlets@1.3.29?type=jar",
        "pkg:maven/io.dropwizard/dropwizard-jetty@1.3.29?type=jar",
        "pkg:maven/io.dropwizard/dropwizard-lifecycle@1.3.29?type=jar",
        "pkg:maven/io.dropwizard.metrics/metrics-core@4.1.16?type=jar",
        "pkg:maven/io.dropwizard.metrics/metrics-jvm@4.1.16?type=jar",
        "pkg:maven/io.dropwizard.metrics/metrics-jmx@4.1.16?type=jar",
        "pkg:maven/io.dropwizard.metrics/metrics-servlets@4.1.16?type=jar",
        "pkg:maven/io.dropwizard.metrics/metrics-healthchecks@4.1.16?type=jar",
        "pkg:maven/io.dropwizard/dropwizard-request-logging@1.3.29?type=jar",
        "pkg:maven/net.sourceforge.argparse4j/argparse4j@0.8.1?type=jar",
        "pkg:maven/org.eclipse.jetty.toolchain.setuid/jetty-setuid-java@1.0.4?type=jar"
      ]
    },

```
In v2.3.0, this changes to:

```
  }, {
    "ref" : "pkg:maven/io.dropwizard/dropwizard-core@1.3.29?type=jar",
    "dependsOn" : [ "pkg:maven/io.dropwizard/dropwizard-util@1.3.29?type=jar", "pkg:maven/io.dropwizard/dropwizard-configuration@1.3.29?type=jar", "pkg:maven/io.dropwizard/dropwizard-metrics@1.3.29?type=jar", "pkg:maven/io.dropwizard/dropwizard-servlets@1.3.29?type=jar", "pkg:maven/io.dropwizard/dropwizard-jetty@1.3.29?type=jar", "pkg:maven/io.dropwizard/dropwizard-lifecycle@1.3.29?type=jar", "pkg:maven/io.dropwizard.metrics/metrics-core@4.1.16?type=jar", "pkg:maven/io.dropwizard.metrics/metrics-jvm@4.1.16?type=jar", "pkg:maven/io.dropwizard.metrics/metrics-jmx@4.1.16?type=jar", "pkg:maven/io.dropwizard.metrics/metrics-servlets@4.1.16?type=jar", "pkg:maven/io.dropwizard.metrics/metrics-healthchecks@4.1.16?type=jar", "pkg:maven/io.dropwizard/dropwizard-request-logging@1.3.29?type=jar", "pkg:maven/net.sourceforge.argparse4j/argparse4j@0.8.1?type=jar", "pkg:maven/org.eclipse.jetty.toolchain.setuid/jetty-setuid-java@1.0.4?type=jar" ]
  }, {
```

**Comment by stevespringett** [comment]
@DarthHater any idea how to solve this? This appears to have been caused by the move to Jackson.

**Comment by DarthHater** [comment]
I can take a gander for sure!

**Comment by davewichers** [comment]
I don't think this is exactly a duplicate of #93 as it affects the XML, not JSON, and it affects the XML in more places than is described in the JSON ticket. If you don't reopen #93, please make sure when this issue is addressed that the XML examples in #93 are fixed as well.

**Comment by DarthHater** [comment]
@davewichers you are correct, it's not an exact duplicate but it's very similar. I am working on both here: https://github.com/CycloneDX/cyclonedx-core-java/pull/84
